R/dataDrivenClusters.R

```diff
@@ -27,35 +27,28 @@ dataDrivenClusters <- function(voxel_df, n_pca = 20, n_umap = 2, n_clust = 2,
   umap_df <- as.data.frame(umap_coords)
 
   #apply kmeans to umap-reduced data
-
   km <- kmeans(umap_df, centers = n_clust, iter.max = 5000)
 
   #obtain the dataframe with xyz coords, UMAP coords, and cluster values
-
   data_df <- cbind(cbind(xyz, umap_coords), km$cluster)
 
   #obtain a dataframe with umap coords and kmeans clusters to make plotting easier
-
   plot_df <- as.data.frame(cbind(umap_coords, km$cluster))
```

Comment:
Can we make the data frame using only `data_df`?

Reply:
I've gone ahead and just extracted the 4:6 columns of `data_df` to form `plot_df`.